public/locales/en.json

```diff
@@ -86,7 +86,7 @@
     "when": "Coming soon",
     "length": "March 2023",
     "type": "Speakers",
-    "description": "We are still exploring your submissions so if you have an idea and you would like to talk about it head over to our CFP"
+    "description": "We are still exploring your submissions so if you have an idea and you would like to talk about it head over to our <link href='https://sessionize.com/opensourceday23/'>CFP</link>"
```

Comment:
Attento qui, credo ci sia da mettere il rich text direttamente sul component in questo caso (per riferimento cerca file con link)